### DISCUSSION TITLE
Does powerdns support forwarding dns requests to upstream doh services?

### DISCUSSION THREAD

**Discussion by ShipeiXu** [PowerDNS/pdns#13893] [Q&A] [answered]
We want to deploy powerdns-recursor on the intranet, and then forward the dns request to the upstream doh server in the form of dns over https.
To avoid problems such as pollution and hijacking. Please tell me if it is currently supported.

**Comment by omoerbeek** [comment, marked as answer]
PowerDNS Recursor does not support forwarding over DoH. You might run a dnsdist that has one or more DoH upstreams.

**Comment by rgacogne** [comment]
Please note that PowerDNS Recursor does support DoT, though.
